tuf/download.py

```diff
@@ -257,14 +231,34 @@ def _download_file(url, required_length, STRICT_REQUIRED_LENGTH=True):
   temp_file = securesystemslib.util.TempFile()
 
   try:
-    # Open the connection to the remote file.  _open_connection() can raise
-    # socket connection exceptions (such as SSLError).  Connection errors of
-    # this kind can be minimized by adjusting the socket timeout in
-    # settings.py.
-    connection = _open_connection(url)
+    # Attach some default headers to every Session.
+    _session.headers.update({
+        # Tell the server not to compress or modify anything.
+        # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding#Directives
+        'Accept-Encoding': 'identity',
+        # The TUF user agent.
+        # TODO: Attach version number, which cannot be easily found right now.
```

Comment:
What does this mean?  Do you mean that TUF's communication protocol should have a version of its own so we might use different versions at different times?  Do we want the version of TUF to be here instead?

Reply:
I think it's just overwriting the User-Agent to something that's not `requests` or `python urrlibX`. It'd be cleaner (and easier to analyze traffic) if we used something like `TUF vX`

Reply:
Correct. Sometimes you want to exactly which piece of software was downloading from your server. To be fair, it should be something like `tuf-{tuf-version-number}-over-requests-{requests-version-number}`.